src/more-infos/more-info-configurator.html

```diff
@@ -94,6 +114,10 @@
 </dom-module>
 
 <script>
+
+const CHECKBOX_TYPES = ['checkbox', 'boolean', 'toggle'];
```

Comment:
We should just allow 1 checkbox type. I like `boolean` because it describes the data type and not about how it should be represented in the UI

Reply:
The 'type' attribute in `stateObj.attibutes.fields[]` is the only way to describe how a field will look in the UI.
That's why we can also pass type 'password' or 'email' and the (string) input get's decorated accordingly. 
Coming from html a intuitive thing would be to set `field['type'] = 'checkbox'`, coming from Python the most intuitive thing to do would be to set  `field['type'] = 'boolean'`

I agree that setting `field['type'] = 'toggle'` actually should render a paper-toggle instead of a paper-checkbox. But falling back to checkbox at least gives the same functionality without extra code.

Reply:
We're adding this functionality in this PR. We should decide on how we want it to look. Do we follow Python? HTML? We need to pick 1 and stick with it. Already allowing a wide range is setting ourselves up for chaos.